src/main/java/uk/gov/hmcts/reform/bulkscanprocessor/entity/EnvelopeRepository.java

```diff
@@ -89,4 +89,12 @@
     int getIncompleteEnvelopesCountBefore(@Param("datetime") LocalDateTime dateTime);
 
     List<Envelope> findByContainerAndStatusAndZipDeleted(String container, Status status, boolean zipDeleted);
+
+    @Query("select e from Envelope e\n"
+        + "where e.zipFileCreateddate between :fromDateTime AND :toDateTime"
+    )
```

Comment:
unfortunately it won't be that simple - if zip file gets rejected envelopes is not created, just an event.